source/validate.cpp

```diff
@@ -287,6 +292,10 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInsts,
 
 namespace {
 
+// Shame
+#define CHECK_RESULT(EXPRESSION) \
```

Comment:
Hmm. I don't know how I missed that.

I find that the macro form is more readable. I can expand it to an if statement if that is not a significant enough motivation.